### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by rafrombrc**
Provide more depth and clarity describing what metric-hub is, including a number of additional links out to supporting docs and resources.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
+
+Some metrics may only be needed within a limited context, such as a value to be tracked over the course of an experiment (handled by [jetstream](https://github.com/mozilla/jetstream), our automated experiment analysis framework), or a piece of data to be ongoing monitored for one or more populations (via [OpMon](https://github.com/mozilla/opmon), an operational monitoring tool used to watch the impact of scaled roll-outs and to support certain product health dashboards). Other metrics are much more widely impactful, such as organizational KPIs and high level product success measurements.
```
Comment (sean-rose) [inline README.md:5]: "ongoing monitored" seems like awkward phrasing.  Maybe "continuously monitored"?

File: README.md [hunk 2]
```diff
@@ -1,12 +1,15 @@
...
-or [opmon](https://github.com/mozilla/opmon).
+All metric definitions can and should be referenced in other tooling and their configurations, including jetstream, OpMon, [BigQuery ETL](https://docs.telemetry.mozilla.org/concepts/metric_hub.html#using-metrics-in-etl-queries), and [python scripts](https://docs.telemetry.mozilla.org/concepts/metric_hub.html#using-metrics-in-python-scripts). All metrics are also made available in auto-generated Looker Explores, from which visualizations and dashboards can be generated.
 
 Jetstream specific configs or defaults can be added in the `jetstream/` directory.
 OpMon specific configs or defaults can be added in the `opmon/` directory.
+Looker Explore specific configs or defaults can be added in the `looker/` directory. This directory also supports the definitions of "statistics" (i.e. specific aggregations of metric-hub metrics), which will be materialized as measures in the autogenerated Explores. How to do this is explained in detail in the [documentation](https://docs.telemetry.mozilla.org/concepts/metric_hub.html#using-metrics-in-looker).
```
Comment (sean-rose) [inline README.md:13]: Making this section a list could help with readability.  As it is all three sentences will get combined into a single paragraph.

Comment (mikewilli) [inline README.md:13]: Suggestion to make the separate pieces more visible:
```suggestion
- **Jetstream** specific configs or defaults can be added in the `jetstream/` directory.
- **OpMon** specific configs or defaults can be added in the `opmon/` directory.
- **Looker Explore** specific configs or defaults can be added in the `looker/` directory. This directory also supports the definitions of "statistics" (i.e. specific aggregations of metric-hub metrics), which will be materialized as measures in the autogenerated Explores. How to do this is explained in detail in the [documentation](https://docs.telemetry.mozilla.org/concepts/metric_hub.html#using-metrics-in-looker).
```

File: README.md [hunk 3]
```diff
@@ -1,12 +1,15 @@
...
 
-Tool-specific configs take precedence over the metric definitions in `definitions/` when used in the tooling, in all other contexts `definitions/` is seen as the source of truth.
+Tool-specific configs take precedence over the metric definitions in the top level `definitions/` folder when used in the tooling, in all other contexts `definitions/` is seen as the source of truth.
```
Comment (sean-rose) [inline README.md:15]:
```suggestion
Tool-specific configs take precedence over the metric definitions in the top level `definitions/` folder when used in the tooling, while in all other contexts `definitions/` is seen as the source of truth.
```

File: README.md [hunk 4]
```diff
@@ -1,12 +1,15 @@
...
+
 For more information on how to add metric definitions please see the [docs on docs.telemetry.mozilla.org](https://docs.telemetry.mozilla.org/concepts/metric_hub.html)
 
-All metric definitions can be referenced in other tooling and their configurations, such as, [jetstream](https://github.com/mozilla/jetstream)
-or [opmon](https://github.com/mozilla/opmon).
+All metric definitions can and should be referenced in other tooling and their configurations, including jetstream, OpMon, [BigQuery ETL](https://docs.telemetry.mozilla.org/concepts/metric_hub.html#using-metrics-in-etl-queries), and [python scripts](https://docs.telemetry.mozilla.org/concepts/metric_hub.html#using-metrics-in-python-scripts). All metrics are also made available in auto-generated Looker Explores, from which visualizations and dashboards can be generated.
```
Comment (sean-rose) [inline README.md:9]:
```suggestion
All metric definitions can and should be referenced in other tooling and their configurations, including jetstream, OpMon, [BigQuery ETL](https://docs.telemetry.mozilla.org/concepts/metric_hub.html#using-metrics-in-etl-queries), and [Python scripts](https://docs.telemetry.mozilla.org/concepts/metric_hub.html#using-metrics-in-python-scripts). All metrics are also made available in auto-generated Looker Explores, from which visualizations and dashboards can be generated.
```

**Review by sean-rose — APPROVED** [review]
r+wc

**Review by mikewilli — APPROVED** [review]
Hooray for doc updates!

Just had one suggestion, and it might help to break it up with some sub-headings if you're feeling extra ambitious about it, but this is already a nice improvement as-is.

**Pull request merged**